client/src/templates/Challenges/components/Side-Panel.js

```diff
@@ -35,7 +33,8 @@ const propTypes = {
 };
 
 export class SidePanel extends Component {
-  componentDidMount() {
+  async componentDidMount() {
+    const MathJax = await global.MathJax;
     if (MathJax) {
```

Comment:
The case where MathJax isn't loaded needs to be handled, too.  This https://github.com/freeCodeCamp/freeCodeCamp/blob/master/client/src/components/Donation/components/DonateModal.js#L55-L61 is how it was handled for Stripe.